src/lib/impl/SubscriberImpl.js

```diff
@@ -212,7 +216,7 @@ class SubscriberImpl extends EventEmitter {
   _disconnectClient(clientId) {
     const client = this._pubClients[clientId];
     if (client) {
```

Comment:
Shouldn't this have an else statement that checks for `this._pendingPubClients[clientId]` ?